sympy/physics/control/lti.py

```diff
@@ -249,6 +261,102 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dc_gain(self):
+        """
+        Computes the low-frequency gain of the frequency response in the
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
```

Comment:
PEP 257 says that there should be a one-line summary followed by a blank line and a longer description

Reply:
One possible option:
```suggestion
        Computes the gain as frequency approaches zero.
        
        For systems with pure integrator terms, the DC gain is infinite.
```
feel free to reword